addons/mail/static/tests/new/message/message_tests.js

```diff
@@ -1103,3 +1103,27 @@ QUnit.test("Toggle star should update starred counter on all tabs", async functi
     await tab1.click(".o-mail-message-action-toggle-star");
     assert.strictEqual(tab2.target.querySelector(".o-starred-box .badge").textContent, "1");
 });
+
+QUnit.test("allow attachment image download on message", async function (assert) {
+    const pyEnv = await startServer();
+    const mailChannelId1 = pyEnv["mail.channel"].create({ name: "test" });
+    const irAttachmentId1 = pyEnv["ir.attachment"].create({
+        name: "Blah.jpg",
+        mimetype: "image/jpeg",
+    });
+    pyEnv["mail.message"].create({
+        attachment_ids: [irAttachmentId1],
+        body: "<p>Test</p>",
+        model: "mail.channel",
+        res_id: mailChannelId1,
+    });
+    const { openDiscuss } = await start({
+        discuss: {
+            context: {
+                active_id: mailChannelId1,
```

Comment:
active id can now be passed to the open discuss function directly. It's less cumbersome to write :)

Reply:
Yes, I forgot :)